_config.yml

```diff
@@ -73,6 +73,11 @@ redirects:
   /ar/إدعم-ألبت-كوين: /ar/support-bitcoin
   /ar/مفردات: /ar/vocabulary
   /ar/تحتاج-أن-تعلم: /ar/you-need-to-know
+  /en/bitcoin-core/2016-01-07-statement: https://bitcoinco.re/en/2016/01/07/statement
+  /en/bitcoin-core/capacity-increase: https://bitcoinco.re/en/2015/12/21/capacity-increase
+  /en/bitcoin-core/capacity-increase-faq: https://bitcoinco.re/en/2015/12/23/capacity-increases-faq
```

Comment:
This is capacity-increase**s**-faq with a s.  All the translated versions are singular.  (Sorry for the inconsistency.)